Shared/Samples/Set surface navigation constraint/SetSurfaceNavigationConstraintView.swift

```diff
@@ -0,0 +1,43 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct SetSurfaceNavigationConstraintView: View {
+    /// The Webscene with a portal item.
+    @State private var scene: ArcGIS.Scene = {
+        // Creates the portal item using the id of the Webscene in the ArcGIS portal.
```

Comment:
```suggestion
        // Creates the portal item using the ID of the web scene in the ArcGIS portal.
```
When uncertain about a proper noun, use [Esri GIS Dictionary](https://support.esri.com/en-us/gis-dictionary/web-map). Here we don't have an entry for "web scene", but it shares the same concepts as web map.